gitpod-io/gitpod
# add dekstop-ide-ready phase to status_rendered analytics event

## Description


To track whether a user seen loading screen after configuring desktop IDE.

## Related Issue(s)

N/A

## How to test


- Start a workspace: https://ak-ide-desktop-ready.staging.gitpod-dev.com/#referrer:jetbrains-gateway/https://github.com/akosyakov/spring-petclinic
- Verify an event in staging untrusted: https://app.segment.com/gitpod/sources/staging_untrusted/debugger
   - You should see `status_rendered` event with `dekstop-ide-ready` phase which has new properties `kind` for IDE kind and `clientID` for a client app to open, e.g. IntelliJ and JetBrains Gateway
<img width="786" alt="Screenshot 2022-02-25 at 09 59 48" src="https://user-images.githubusercontent.com/3082655/155686100-d50cc1ae-af84-4ce3-8de8-df2588a520d3.png">

## Release Notes

```release-note
NONE
```

## Documentation


/werft analytics=segment|TEZnsG4QbLSxLfHfNieLYGF4cDwyFWoe
